parsely_raw_data/__init__.py

```diff
@@ -28,3 +29,39 @@
     'stream',
     'utils',
 ]
+
+BOOLEAN_FIELDS = [
+    "flags_is_amp",
+]
+
+
+def normalize_keys(r, schema):
+    """Conform events to public schema: correct keys and proper value types."""
+    schema = schema or schema.SCHEMA
+    event_dict = {}
+    version =__version__
+
+    # fix value types
+    if r.get("metadata.share_urls") is not None and isinstance(
+        r["metadata.share_urls"], dict
+    ):
+        r["metadata.share_urls"] = list(r["metadata.share_urls"].values()) or None
+
+    # emit only public schema items
+    for key, val in iteritems(r):
+        key = key.replace(".", "_")
+        if key in schema:
+            event_dict[key] = val
+
+    # ensure all columns are available and null when needed
+    # account for all boolean schema defined fields as this is parsely_raw_data specific
+    for key in schema:
+        if key not in event_dict.keys():
+            if key in BOOLEAN_FIELDS:
+                event_dict[key] = False
+            else:
+                event_dict[key] = None
+
+    event_dict["schema_version"] = version
```

Comment:
If you're only going to use `version` in one place, seems like this could be `= __version__` and avoid allocating a new variable.